### DISCUSSION TITLE
Can't connect to t-pot

### DISCUSSION THREAD

**Discussion by 00ihsan** [telekom-security/tpotce#1312] [Q&A]
So i have one lab subnet and one main subnet. The firewall is configured to accept main to lab but no lab to lan (new connections). I can reach everything on my lab thru this but when i try to access tpot over https on port 64294 it won't load and when i nmap port 64294 i get filtered. Is there a setting within tpot to enable other subnets?

**Comment by 00ihsan** [comment]
Worth noting that SSH (64295) also does not work.

**Reply by gregoryschouweilerLSC** [reply]
I have same problem to acces from another network to management port. There is nothing in documentation to allow a network.
